pkg/volume/glusterfs/glusterfs.go

```diff
@@ -255,8 +260,19 @@ func (b *glusterfsMounter) setUpAtInternal(dir string) error {
 	for hostIP := range addr {
 		errs = b.mounter.Mount(hostIP+":"+b.path, dir, "glusterfs", options)
 		if errs == nil {
+			glog.Infof("glusterfs: successfully mounted %s", dir)
 			return nil
 		}
 	}
+
+	// Failed mount scenario.
+	// Since gluster does not return eror text
+	// it all goes in a log file, we will read the log file
+	logerror := readGlusterLog(log, b.pod.Name)
+	if logerror != nil {
+		glog.Errorf("glusterfs: mount failed: %v", logerror)
+		return fmt.Errorf("glusterfs: mount failed: %v", logerror)
```

Comment:
@rootfs - although I do see some value in still providing the log file path to the user I guess, if nothing else for reference and history...and if we add some more verbiage like you suggested, it might be useful, let me try it out and I'll update this PR with the new results